src/index.js

```diff
@@ -45,7 +45,7 @@ const EcomApps = function (ecomAuth = new EcomAuth()) {
   this.findApp = appId => findApp(appId)
   this.installApp = (appId, redirect, body) => installApp(self, appId, redirect, body)
   this.removeApplication = _id => removeApplication(self, _id)
-  this.editApplication = (_id, body) => editApplication(self, _id, body)
+  this.editApplication = (_id, body, updateInternals=false) => editApplication(self, _id, body, updateInternals)
```

Comment:
```suggestion
  this.editApplication = (_id, body, canUpdateInternals) => editApplication(self, _id, body, canUpdateInternals)
```

Só minors aqui, ideal é começar booleano sempre com `is`, `can` ou `has` (tudo bem que aqui não criamos um contrato o argumento não é um objeto desconstruído), e o valor padrão já é setado diretamente na função, não precisamos repetí-la aqui (DRY).